templates/auth/login.html

```diff
@@ -39,8 +39,6 @@ <h5>Login required</h5>
         <div class="grid-x">
             <div class="medium-6 columns">
                 {{ macros.render_field(form.email) }}
```

Comment:
Removing the 2 columns is no good.

Reply:
I'm not in favour of this. It's always been like that, and makes just as much sense to me as to have them below another. Please revert.